integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/MetalakePageTest.java

```diff
@@ -4,14 +4,133 @@
  */
 package com.datastrato.gravitino.integration.test.web.ui;
 
+import com.datastrato.gravitino.integration.test.web.ui.Pages.MetalakePage;
 import com.datastrato.gravitino.integration.test.web.ui.utils.AbstractWebIT;
 import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
 
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 public class MetalakePageTest extends AbstractWebIT {
+  MetalakePage metalakePage = new MetalakePage(driver);
+
   @Test
   public void homePage() {
     String title = driver.getTitle();
     Assertions.assertEquals("Gravitino", title);
   }
+
+  @Test
+  @Order(1)
+  public void testCreateMetalake() {
+    metalakePage.createMetalakeAction("test", "test", true);
+
+    boolean status = metalakePage.verifyIsCreatedMetalake();
+
+    if (status) {
+      LOG.info("create metalake successful");
```

Comment:
We don't have to do the log in  java test. Just check the return value is expected or not is enough. @xunliu please help @ch3yne to make this test more java-ish.